src/libcprover-rust/src/c_api.cc

```diff
@@ -59,3 +59,33 @@ std::vector<std::string> const &get_messages()
 {
   return output;
 }
+
+verifier_resultt get_verification_result(
```

Comment:
Why has this file being touched by this PR?
It shouldn't be changed.

Reply:
removed, sorry for cluttering